crud/src/main/java/com/redhat/lightblue/rest/crud/AbstractCrudResource.java

```diff
@@ -309,8 +309,9 @@ public Response simpleFind(@PathParam(PARAM_ENTITY) String entity,
                                @QueryParam("P") String p,
                                @QueryParam("S") String s,
                                @QueryParam("from") Long from,
-                               @QueryParam("to") Long to) throws IOException {
-        return simpleFind(entity, null, q, p, s, from, to);
+                               @QueryParam("to") Long to,
+                               @QueryParam("maxResults") Long maxResults) throws IOException {
+        return simpleFind(entity, null, q, p, s, from, to, maxResults);
```

Comment:
This list seems to be growing.  Does it make sense to replace this list with `@BeanParam`?

https://jersey.java.net/apidocs/latest/jersey/javax/ws/rs/BeanParam.html